### PR TITLE
Add getter and setter for each command line option

### PR DESCRIPTION
**Pull request by melissalinkert**
Corresponds to https://github.com/glencoesoftware/bioformats2raw/pull/190, see also https://github.com/glencoesoftware/NGFF-Converter/issues/39.

### CODE REVIEW

**Review by sbesson — COMMENTED** [review]
See https://github.com/glencoesoftware/bioformats2raw/pull/190#pullrequestreview-1338923504 for a first pass on the review of this new API

**Comment by melissalinkert** [comment]
399f418 has the input/output path changes mentioned in https://github.com/glencoesoftware/bioformats2raw/pull/190, but I'm still working on a better way to set the compression type so that isn't ready for re-testing yet.

**Comment by melissalinkert** [comment]
@sbesson: 0344585 was the last planned change I had here, so this is ready for review together with the current state of https://github.com/glencoesoftware/bioformats2raw/pull/190.

**Review by sbesson — APPROVED** [review]
One comment about the max workers validation to match the behavior of bioformats2raw. Otherwise, tested ``--log-level`, `-p`, `--compression`, `--quality` `--rgb` and the behavior of the utility remains unchanged.

File: src/main/java/com/glencoesoftware/pyramid/PyramidFromDirectoryWriter.java [hunk 1]
```diff
@@ -115,93 +116,273 @@ public class PyramidFromDirectoryWriter implements Callable<Void> {
...
+   *
+   * @param workers maximum worker count
+   */
   @Option(
       names = "--max_workers",
       description = "Maximum number of workers (default: ${DEFAULT-VALUE})"
   )
-  int maxWorkers = Runtime.getRuntime().availableProcessors();
+  public void setMaxWorkers(int workers) {
+    maxWorkers = workers;
```
[the remark this answers is not in the document]
Comment (melissalinkert) [reply src/main/java/com/glencoesoftware/pyramid/PyramidFromDirectoryWriter.java]: Done in dda535a.

**Review by sbesson — APPROVED** [review]

**Pull request merged**